CodeStarterCamp_Week2/randomLottoNumber.swift

```diff
@@ -8,16 +8,37 @@
 import Foundation
 
 // 로또 당첨 번호를 생성하는 함수
-func makeLottoNumbers() -> Set<Int> {
+func makeLottoNumbers() {
     var lottoNumbers: Set<Int> = Set<Int>()
     var lottoNumber: Int
     while lottoNumbers.count < 6 {
         lottoNumber = Int.random(in: 1...45)
         lottoNumbers.insert(lottoNumber)
     }
-    return lottoNumbers
+    // 생성한 로또 당첨 번호를 저장
+    saveLottoNumbers(newLottoNumbers: lottoNumbers)
 }
 
+// 로또 당첨 번호를 저장하는 함수
+func saveLottoNumbers(newLottoNumbers: Set<Int>) {
+    let turn = lottoHistory.count
+    lottoHistory[String(turn+1)+"회차"] = newLottoNumbers
```

Comment:
회차를 Dictionary count에 맞게 잘 생성해주고 있네요 👍
가독성을 위해 공백 컨벤션도 신경써주시면 좋겠습니다 ~!

```suggestion
    let turn = lottoHistory.count
    lottoHistory[String(turn + 1) + "회차"] = newLottoNumbers
```